src/counter.js

```diff
@@ -1,13 +1,23 @@
-export function createCounter() {
+export function createCounter(initObject = {}) {
```

Comment:
```suggestion
export function createCounter({min, max, value} = {}) {
```
비구조화 할당을 쓰면 매번마다 `.` 연산을 쓰지 않아도 되서 번거로움이 적을 것같아요!